docs/source/developer-docs/contrib_guide_code.rst

```diff
@@ -39,6 +40,12 @@ below to get started. If you experience any problems while working on the issue,
 leave a comment on the issue page in GitHub and someone on the core team will
 be able to lend you assistance.
 
+Keep in mind that all you see below is just a guide, that describe how things
```

Comment:
Please keep in mind that what follows are guidelines. If you work through the steps and have questions or run into time constraints, please submit what you already have worked on as a pull request and ask questions on it. Your effort, including partial or in-progress work, is appreciated.